test/fixtures/sitemap.xml

```diff
@@ -3,9 +3,9 @@
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
    <url>
-       <loc>https://hex.pm/packages/postgrex</loc>
-       <lastmod>2014-04-17T14:00:00Z</lastmod>
-       <changefreq>daily</changefreq>
-       <priority>0.8</priority>
+      <loc>https://hex.pm/packages/postgrex</loc>
+      <lastmod>2014-04-17T14:00:00Z</lastmod>
+      <changefreq>daily</changefreq>
+      <priority>0.8</priority>
```

Comment:
yeah, it was 3 spaces in 1st level, and 4 spaces in 2nd level of indent, so I just unified it to be 3 spaces everywhere. I exchanged comments with @lexmag about these indentations and I really don't care which way it's gonna be, but I thought it should be unified. I can revert these changes too